WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -315,6 +316,9 @@ public void run() {
             GCMRegistrationIntentService.enqueueWork(this,
                     new Intent(this, GCMRegistrationIntentService.class));
         }
+
+        AppRatingDialog.INSTANCE.init(this);
+        AppRatingDialog.INSTANCE.showRateDialogIfNeeded(getFragmentManager());
```

Comment:
We probably don't want to show the rating dialog if the logic above caused the login or signup epilogues to appear. I also wonder if we should skip it if the user opened the app from a push notification or a shortcut. The reasoning is that in those situations the user had a specific task in mind and may be annoyed at being interrupted by a rating dialog.

Reply:
Good idea! There is a lot of stuff going on in `onCreate`, so I updated the code to only show the dialog when the user launches app "normally".